### ISSUE TITLE
Chapter Calendar

### ISSUE THREAD

**Issue by bethschechter** [maptime/maptime-admin#52]
It would be amazing to see when chapters were meeting all around the world. 
Is there an easy way to do this?
If not, that's ok, but I'd be interested to find out.

**Comment by almccon** [comment]
For Maptimes that use Meetup, there's a clunky way I found to get Meetup.com to show all the different meetups on the calendar. And I think we could have somebody write something with the Meetup API to do that for real. But that doesn't catch everybody.

To catch all the chapters, we'd need them all to enter their information into a calendar somewhere.

**Comment by bethschechter** [comment]
Yeah, I don't know if this will actually work. But it was on the HQ asana,
so I migrated it. Definitely loooow priority at the moment

On Fri, Apr 24, 2015 at 8:32 PM, Alan McConchie <notifications@github.com>
wrote:

> For Maptimes that use Meetup, there's a clunky way I found to get
> Meetup.com to show all the different meetups on the calendar. And I think
> we could have somebody write something with the Meetup API to do that for
> real. But that doesn't catch everybody.
>
> To catch all the chapters, we'd need them all to enter their information
> into a calendar somewhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maptime/maptime-admin/issues/52#issuecomment-96124585>
> .
>



-- 
@bethschechter
Maptime <http://maptime.io/> founder
ATL –> SF –> PDX

**Comment by geobrando** [comment]
See:
maptime/maptime.github.io#105
maptime/maptime.github.io#192
